R-package/R/lgb.Booster.R

```diff
@@ -625,6 +625,9 @@ Booster <- R6Class(
 #' 
 #' @examples
 #' \dontrun{
+#' # It crashes the documentation generation, skipping
+#' if (!(pkgdown::in_pkgdown())) {
```

Comment:
why these crash ?

Reply:
It is because I used the version without https://github.com/Microsoft/LightGBM/commit/ffeae5a660147b7b04a9e7cd399953af271a9511

I can now remove the crash workarounds with your fix.